Makefile

```diff
@@ -21,6 +21,24 @@ BENCH_RUN=NONE
 test:
 	$(GOTEST) -run=$(TEST_REGEX) -count=$(TEST_COUNT) ./...
 
+# lint runs all local linters that must pass before pushing
+.PHONY: lint lint-install lint-fast
+lint: lint-install
+	goimports -w .
+	gofmt -w .
+	faillint -ignore-tests -paths "$(shell grep -v '^#' .faillint | xargs echo | sed 's/ /,/g')" ./...
+	go mod tidy
```

Comment:
I don't think `go mod tidy` is part of linting actions but if you add it, I think it should be run before linters or I think linters will download stuff.